spec/app/ics-028-cross-chain-validation/system_model_and_properties.md

```diff
@@ -169,7 +169,7 @@ As a result, it is possible for multiple VSCs to be received by a consumer chain
 i.e., the validator updates within the VSCs are being *aggregated* by keeping only the latest update per validator. 
 As a consequence, some validator sets on the provider chain are not existing on all consumer chains. 
 In other words, the validator sets on each consumer chain form a *subsequence* of the validator sets on the provider chain. 
-Nonetheless, as a **requirement of CCV**, *all the validator updates on the provider chain MUST be included in the sequence of validator sets on all consumer chains*.
+Nonetheless, as a **requirement of CCV**, *all the validator sets on all the consumer chains MUST be included in the sequence of validator sets on the provider chain*.
```

Comment:
The idea here is to make sure that all validator updates that occur on the **_provider chain_** are provided to the consumer chain. Let two updates `U1` and `U2` to the same validator `V` occur in two subsequent blocks on the provider, i.e., 
- `V` has 100 ATOMs bonded
- `U1`: V's stake changes to 80 (a delegator undelegates 20 atoms)
- `U2`: V's stake changes to 50 (a delegator undelegates 30 atoms)

The provider chain sends `U1` and `U2` to the consumer chain, however, due to delays, it may be that both are received by a consumer chain in the same block. Thus, `V`'s power on that consumer chain will go from 100 directly to 50 (I assume here that's a 1 to 1 conversion between stake and power). Basically, `U1` is included in `U1` (see the Validator Update Inclusion property below). 

The point of this sentence is to express that the effect of `U1` is not lost from the consumer chain's perspective.

Reply:
I see, I was just confused by the wording. I will suggest another wording.